_includes/supporter.html

```diff
@@ -10,9 +10,12 @@
       and maintain the inventory for the conference.
     </div>
     <p class="description">
-      Reach out to us at
-      <a href="mailto:sponsor@in.pycon.org">sponsor@in.pycon.org</a>
+      Take a look at
+      <a href="{{ '/sponsorship-prospectus' | relative_url }}">Sponsorship Page</a>
       to support PyCon India 2020.
     </p>
+    <div class="text-center" style="position: relative;">
+      <a href="/2020/sponsorship-prospectus.pdf" class="dwd-link"><b>DOWNLOAD SPONSORSHIP PROSPECTUS</b></a>
```

Comment:
How about having this as a button?

```suggestion
      <a class="primary-button" rel="nofollow noreferrer" href="/2020/sponsorship-prospectus.pdf" target="_blank">
        Sponsorship Prospectus
      </a>
```